modules/rosapps/applications/net/ncftp/ncftp/main.c

```diff
@@ -24,7 +24,7 @@
 	WSADATA wsaData;
 	int wsaInit = 0;
 
-	__inline void DisposeWinsock(int aUNUSED) { if (wsaInit > 0) WSACleanup(); wsaInit--; }
+	static __inline void DisposeWinsock(int aUNUSED) { if (wsaInit > 0) WSACleanup(); wsaInit--; }
```

Comment:
Is the underscored inline here correct? (since you removed these in a previous commit)

Reply:
I'm not even sure this is our code...